pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -694,7 +696,7 @@ func parseSystemAdapterConfig(config *zconfig.EdgeDevConfig,
 			if err != nil {
 				log.Errorf("parseSystemAdapterConfig: Network with UUID %s not found: %s\n",
 					sysAdapter.NetworkUUID, err)
-				return
+				continue
```

Comment:
AFAICT this change is no longer needed, and it would make sense to undo it. So that we do push through the updates to the interfaces which we know about.

Your other changes fixes the root cause.

Reply:
My bad; I read backwards.